src/components/enterprise-user-subsidy/enterprise-offers/data/constants.js

```diff
@@ -12,4 +15,8 @@ export const ENTERPRISE_OFFER_STATUS = {
   CONSUMED: 'Consumed',
 };
 
-export const ENTERPRISE_OFFER_LOW_BALANCE_THRESHOLD = 200;
+export const ENTERPRISE_OFFER_LOW_BALANCE_THRESHOLD = 200; // dollar value amount threshold
```

Comment:
[curious] Were we going to use $100 here?

Reply:
yep! forgot to push the changes.